scripts/install-argocd.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Find the latest ArgoCD version
+VERSION=$(curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
+echo $VERSION
+
+# Platform check
+PLATFORM=`uname -s | awk '{print tolower($0)}'`
+echo $PLATFORM 
+
+# Fetch the appropriate OS binary for ArgoCD
+curl -sSL -o argocd https://github.com/argoproj/argo-cd/releases/download/$VERSION/argocd-$PLATFORM-amd64
+
+# Make the argocd CLI executable
+chmod +x argocd
```

Comment:
```chmod +x bin/argocd```

Reply:
For the `install-argocd.sh` script, the binary is downloaded into current directory so shouldn't the chmod be executed on the filename for the working directory and not `bin/...`? argocd binary is not under `bin/` for the context of `install-argocd.sh` script.

Reply:
As the pr #69 has been merged, so we need to download the argocd binary into ```bin/``` directory like i mentioned in https://github.com/redhat-developer/kam/pull/71#discussion_r511072998 with the reason. So the execute permission will be applied on ```bin/argocd```

Reply:
Thanks for the explanation @amitkrout

Reply:
```
 chmod: cannot access 'argocd': No such file or directory
error: failed to execute wrapped command: exit status 1 
```
Details log - https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-developer_kam/71/pull-ci-redhat-developer-kam-master-v4.5-integration-e2e/1320186431414472704#1:build-log.txt%3A905

@dewan-ahmed Please update the path here too. For example 
```chmod +x bin/argocd```